--- XFree86-fonts-4.1.0/xc/fonts/scaled/Type1/Imakefile~	Fri Mar 30 04:15:18 2001
+++ XFree86-fonts-4.1.0/xc/fonts/scaled/Type1/Imakefile	Sat Aug 18 03:00:18 2001
@@ -26,14 +26,12 @@
 
 MakeFontsDir(fonts.scale)
 #if UseFreeType2
-MakeXftCache($(FONTS))
 #endif
 
 InstallFonts($(FONTINSTDIR))
 InstallFontScale($(FONTINSTDIR))
 InstallEncodingsDir($(ENCODINGSSRC),$(FONTINSTDIR))
 #if UseFreeType2
-InstallXftCache($(FONTINSTDIR))
 #endif
 
 #ifdef IHaveSubdirs
