diff -urN xc.orig/fonts/scaled/TTF/Imakefile xc/fonts/scaled/TTF/Imakefile
--- xc.orig/fonts/scaled/TTF/Imakefile	Sun Mar  9 13:47:53 2003
+++ xc/fonts/scaled/TTF/Imakefile	Sun Mar  9 14:47:49 2003
@@ -11,13 +11,11 @@
 MakeFontsDir(fonts.scale)
 
 #if UseFreetype2
-MakeFcCache($(FONTS))
 #endif
 
 InstallFonts($(FONTINSTDIR))
 InstallEncodingsDir($(ENCODINGSSRC),$(FONTINSTDIR))
 #if UseFreetype2
-InstallFcCache($(FONTINSTDIR))
 #endif
 
 #ifdef IHaveSubdirs
diff -urN xc.orig/fonts/scaled/Type1/Imakefile xc/fonts/scaled/Type1/Imakefile
--- xc.orig/fonts/scaled/Type1/Imakefile	Sun Mar  9 13:47:53 2003
+++ xc/fonts/scaled/Type1/Imakefile	Sun Mar  9 14:25:27 2003
@@ -32,13 +32,11 @@
 MakeFontsScale($(OBJS))
 MakeFontsDir(fonts.scale)
 #if UseFreetype2
-MakeFcCache($(FONTS))
 #endif
 
 InstallFonts($(FONTINSTDIR))
 InstallEncodingsDir($(ENCODINGSSRC),$(FONTINSTDIR))
 #if UseFreetype2
-InstallFcCache($(FONTINSTDIR))
 #endif
 
 #ifdef IHaveSubdirs
--- xc/fonts/scaled/Ethiopic/Imakefile.orig	2003-02-04 03:18:19.000000000 +0100
+++ xc/fonts/scaled/Ethiopic/Imakefile	2003-06-04 00:53:08.000000000 +0200
@@ -7,7 +7,6 @@
 OTFINSTDIR = $(FONTDIR)/OTF
 
 #if UseFreetype2
-MakeFcCache($(TTFFONTS) $(OTFFONTS))
 #endif
 
 InstallMultipleFlags($(TTFFONTS),$(TTFINSTDIR),$(INSTDATFLAGS))
--- xc/fonts/scaled/Meltho/Imakefile.orig	2003-06-04 01:27:53.000000000 +0200
+++ xc/fonts/scaled/Meltho/Imakefile	2003-06-04 01:28:04.000000000 +0200
@@ -15,13 +15,11 @@
 MakeFontsDir(fonts.scale)
 
 #if UseFreetype2
-MakeFcCache($(FONTS))
 #endif
 
 InstallFonts($(FONTINSTDIR))
 InstallEncodingsDir($(ENCODINGSSRC),$(FONTINSTDIR))
 #if UseFreetype2
-InstallFcCache($(FONTINSTDIR))
 #endif
 
 #ifdef IHaveSubdirs
