diff -ur XFree86-fonts-4.0.2.orig/xc/fonts/bdf/misc/Imakefile XFree86-fonts-4.0.2/xc/fonts/bdf/misc/Imakefile
--- XFree86-fonts-4.0.2.orig/xc/fonts/bdf/misc/Imakefile	Wed Dec 13 00:57:28 2000
+++ XFree86-fonts-4.0.2/xc/fonts/bdf/misc/Imakefile	Sat Dec 30 03:39:42 2000
@@ -230,7 +230,8 @@
 	FontObj(gb24st)
 
 OBJS = $(UCSOBJS) $(DERIVEDOBJS) $(DERIVEDOBJS2) $(PREDERIVEDOBJS) \
-	$(MISCOBJS) $(ARABIC24OBJS) $(CLEARLYUOBJS)
+	$(MISCOBJS) FontObj(arabic24) FontObj(cu-alt12) FontObj(cu-arabic12) \
+	FontObj(cu-lig12) FontObj(cu-pua12) FontObj(cu12)
 
 FONTINSTDIR = $(FONTDIR)/misc
 
@@ -636,14 +637,6 @@
 FontTarget(cu-lig12)
 FontTarget(cu-pua12)
 FontTarget(cu12)
-
-LinkSourceFile(arabic24.bdf,$(ARABIC24DIR))
-
-LinkSourceFile(cu-alt12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-arabic12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-lig12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-pua12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu12.bdf,$(CLEARLYUDIR))
 
 #if TruncateUCSFonts
 MakeTruncatedUCSBdfFont(5x7,MaxUCSChar,5x7t)
