diff -urN XFree86-ISO8859-2/Makefile XFree86-ISO8859-2.pld/Makefile
--- XFree86-ISO8859-2/Makefile	Thu Jan  1 01:00:00 1970
+++ XFree86-ISO8859-2.pld/Makefile	Thu May  6 22:20:46 1999
@@ -0,0 +1,49 @@
+FONTDIR=/usr/share/fonts/ISO8859-2
+
+#
+# Select whether or not you want to compress the fonts and which compression
+# utility to use for that
+COMPFONTS=set
+COMPUTIL=gzip
+COMPFLAGS=-9f
+
+FONTC=bdftopcf
+FONTCFLAGS=
+MKFONTDIR=mkfontdir
+MKFONTDIRFLAGS=
+
+FONTS=misc 75dpi 100dpi
+
+all: $(addsuffix .done, $(FONTS))
+
+%.done: %
+	cd $< ;\
+	for i in *.bdf ; do\
+	   $(FONTC) $(FONTCFLAGS) $$i > `basename $$i .bdf`.pcf;\
+	   if [ $(COMPFONTS) ];then\
+	      $(COMPUTIL) $(COMPFLAGS) `basename $$i .bdf`.pcf;\
+	   fi;\
+	done;\
+	$(MKFONTDIR) $(MKFONTDIRFLAGS);
+
+install: installfonts
+
+installfonts:
+	for i in $(FONTS) ;do\
+	   cd $$i;\
+	   if [ -f fonts.dir ] ; then\
+	      install -d $(FONTDIR)/$$i;\
+	      install *.pcf* $(FONTDIR)/$$i;\
+	      install fonts.dir $(FONTDIR)/$$i;\
+	      install fonts.alias $(FONTDIR)/$$i;\
+	   fi;\
+	   cd ..;\
+	done;
+
+clean:
+	for i in $(FONTS) ;do\
+	   cd $$i;\
+	   rm -f *.pcf* fonts.dir;\
+	   cd ..;\
+	done;
+
diff -urN XFree86-ISO8859-2/ulT1mo-beta-1.0/Makefile XFree86-ISO8859-2.pld/ulT1mo-beta-1.0/Makefile
--- XFree86-ISO8859-2/ulT1mo-beta-1.0/Makefile	Thu Jan  1 01:00:00 1970
+++ XFree86-ISO8859-2.pld/ulT1mo-beta-1.0/Makefile	Thu May  6 22:22:16 1999
@@ -0,0 +1,24 @@
+FONTDIR=/usr/share/fonts
+
+all: 
+	for j in *.pfb ; do \
+	   mv $$j `basename $$j .pfb`-lat2.pfb; \
+	done; \
+	cd afm; \
+	for j in *.afm ; do \
+	   mv $$j `basename $$j .afm`-lat2.afm; \
+	done; \
+	cd ../pfm; \
+	for j in *.pfm ; do \
+	   mv $$j `basename $$j .pfm`-lat2.pfm; \
+	done;
+
+install:
+	install -d $(FONTDIR)/Type1
+	install -d $(FONTDIR)/Type1/afm
+	install -d $(FONTDIR)/Type1/pfm
+	install *.pfb $(FONTDIR)/Type1
+	install afm/*.afm $(FONTDIR)/Type1/afm
+	install pfm/*.pfm $(FONTDIR)/Type1/pfm
+
+
