diff -urN xc.orig/fonts/bdf/misc/Imakefile xc/fonts/bdf/misc/Imakefile
--- xc.orig/fonts/bdf/misc/Imakefile	Sun Mar  9 13:47:46 2003
+++ xc/fonts/bdf/misc/Imakefile	Sun Mar  9 13:51:42 2003
@@ -10,14 +10,12 @@
 
 #if BuildArabicFonts
 ARABIC24OBJS = FontObj(arabic24)
-ARABIC24DIR = $(TOP)/extras/fonts/arabic24
 #endif
 
 CLEARLYUOBJS = FontObj(cu-alt12) FontObj(cu-arabic12) \
 	FontObj(cu-devnag12) FontObj(cu-lig12) \
 	FontObj(cu-pua12) FontObj(cu12) FontObj(cuarabic12) \
 	FontObj(cudevnag12)
-CLEARLYUDIR = $(TOP)/extras/fonts/ClearlyU
 
 #if BuildJapaneseFonts
 UCSJAOBJS = FontObj(12x13ja) FontObj(18x18ja)
@@ -849,20 +847,6 @@
 FontTarget(gb24st)
 #endif
 
-#if BuildUCSFonts
-#if BuildArabicFonts
-LinkSourceFile(arabic24.bdf,$(ARABIC24DIR))
-#endif
-LinkSourceFile(cu-alt12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-arabic12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-devnag12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-lig12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu-pua12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cu12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cuarabic12.bdf,$(CLEARLYUDIR))
-LinkSourceFile(cudevnag12.bdf,$(CLEARLYUDIR))
-#endif
-
 #if BuildUCSFonts && TruncateUCSFonts
 MakeTruncatedUCSBdfFont(4x6,MaxUCSChar,4x6t)
 MakeTruncatedUCSBdfFont(5x7,MaxUCSChar,5x7t)
