--- XFree86-fonts-4.3.0/xc/fonts/bdf/75dpi/Imakefile.inc.orig	2003-06-03 23:47:47.000000000 +0200
+++ XFree86-fonts-4.3.0/xc/fonts/bdf/75dpi/Imakefile.inc	2003-06-03 23:57:46.000000000 +0200
@@ -78,10 +78,17 @@
 	FontObj(charBI14) FontObj(charBI18) FontObj(charBI24) \
 	FontObj(charI08) FontObj(charI10) FontObj(charI12) FontObj(charI14) \
 	FontObj(charI18) FontObj(charI24) \
+	FontObj(charI08-ISO8859-2) FontObj(charI10-ISO8859-2) \
+	FontObj(charI12-ISO8859-2) FontObj(charI14-ISO8859-2) \
+	FontObj(charI18-ISO8859-2) FontObj(charI24-ISO8859-2) \
 	FontObj(charR08) FontObj(charR10) FontObj(charR12) FontObj(charR14) \
 	FontObj(charR18) FontObj(charR24) \
+	FontObj(charR08-ISO8859-2) FontObj(charR10-ISO8859-2) \
+	FontObj(charR12-ISO8859-2) FontObj(charR14-ISO8859-2) \
+	FontObj(charR18-ISO8859-2) FontObj(charR24-ISO8859-2) \
 	FontObj(tech14) FontObj(techB14) \
-	FontObj(term14) FontObj(termB14)
+	FontObj(term14) FontObj(termB14) \
+	FontObj(term14-ISO8859-2) FontObj(termB14-ISO8859-2)
 
 #if BuildISO8859_1Fonts
 /*
@@ -1312,16 +1319,30 @@
 FontTarget(charI14)
 FontTarget(charI18)
 FontTarget(charI24)
+FontTarget(charI08-ISO8859-2)
+FontTarget(charI10-ISO8859-2)
+FontTarget(charI12-ISO8859-2)
+FontTarget(charI14-ISO8859-2)
+FontTarget(charI18-ISO8859-2)
+FontTarget(charI24-ISO8859-2)
 FontTarget(charR08)
 FontTarget(charR10)
 FontTarget(charR12)
 FontTarget(charR14)
 FontTarget(charR18)
 FontTarget(charR24)
+FontTarget(charR08-ISO8859-2)
+FontTarget(charR10-ISO8859-2)
+FontTarget(charR12-ISO8859-2)
+FontTarget(charR14-ISO8859-2)
+FontTarget(charR18-ISO8859-2)
+FontTarget(charR24-ISO8859-2)
 FontTarget(tech14)
 FontTarget(techB14)
 FontTarget(term14)
 FontTarget(termB14)
+FontTarget(term14-ISO8859-2)
+FontTarget(termB14-ISO8859-2)
 
 #if BuildISO8859_1Fonts
 /*
--- XFree86-fonts-4.3.0/xc/fonts/bdf/misc/Imakefile.orig	2003-06-03 23:12:12.000000000 +0200
+++ XFree86-fonts-4.3.0/xc/fonts/bdf/misc/Imakefile	2003-06-04 00:00:33.000000000 +0200
@@ -327,7 +327,8 @@
 
 MISCOBJS = \
 	$(MISC_8859_1_OBJS) $(MISC_JISX0201_OBJS) $(MISC_JA_OBJS) \
-	$(MISC_KO_OBJS) $(MISC_ZH_OBJS) $(MISC_MISC_OBJS)
+	$(MISC_KO_OBJS) $(MISC_ZH_OBJS) $(MISC_MISC_OBJS) \
+	FontObj(12x24-ISO8859-2) FontObj(8x16-ISO8859-2)
 	
 OBJS = $(UCSOBJS) $(DERIVEDOBJS) $(DERIVEDOBJS2) $(PREDERIVEDOBJS) \
 	$(MISCOBJS) $(ARABIC24OBJS) $(CLEARLYUOBJS)
@@ -787,6 +788,8 @@
 FontTarget(8x16)
 FontTarget(12x24)
 #endif
+FontTarget(8x16-ISO8859-2)
+FontTarget(12x24-ISO8859-2)
 #if BuildJISX0201Fonts
 FontTarget(8x16rk)
 FontTarget(12x24rk)
